BraintreePSP/BraintreePaymentNonceProvider.swift

```diff
@@ -88,7 +97,7 @@ final class BraintreePaymentNonceProvider: PaymentNonceProvider {
             return
         }
 
-        guard self.sdkToken != nil else {
+        guard let authToken = self.sdkToken else {
```

Comment:
FYI: the Swift 5.7 syntax is usable again, so we can go for `guard let sdkToken` for instance.